src/formio.common.ts

```diff
@@ -31,6 +31,36 @@ export interface FormioForm {
   submissionAccess?: AccessSetting[];
 }
 
+export interface GridColumn {
+  label: string;
+  path: string;
+  renderCell?(cellValue: any, component?: any): string;
+}
+
+export interface ComponentInstance {
+  component: ExtendedComponentSchema;
+  getView: any;
```

Comment:
`getView` is not `any`, is a function with parameters. Please reference formio.js library to see what arguments it takes and provide them in the interface